xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -535,6 +553,9 @@ class CPVRRecording final : public CVideoInfoTag
   std::string m_strProviderName; /*!< name of the provider this recording is from */
   int m_iClientProviderUniqueId =
       PVR_PROVIDER_INVALID_UID; /*!< provider uid associated with this recording on the client */
+  int m_ageRating = 0; /*!< parental age rating */
```

Comment:
`int m_ageRating{0};` => coding guidelines

Should this be `unsigned int`? Or can there be negative values?